enterprise/views.py

```diff
@@ -909,7 +915,11 @@ def form_valid(self, form):
                 self.request.user.username,
                 enterprise_customer,
             )
-            return JsonResponse({})
+            response = JsonResponse({})
+            return user_authn_cookies.set_logged_in_cookies(
```

Comment:
… I think we also need to handle the cases where the multiple enterprise selection page is not displayed … e.g. SAML based implicit selection of enterprise

Reply:
in case of SSO flow it is already handled [here](https://github.com/edx/edx-platform/blob/26891841e4063e7878311536c25182d0f705eb7c/common/djangoapps/third_party_auth/pipeline.py#L695)